### PR TITLE
Add brick to disk linking

### PR DESCRIPTION
**Pull request by anmolbabu**
tendrl-bug-id: Tendrl/node-agent#606
Signed-off-by: anmolbabu <anmolbudugutta@gmail.com>

### CODE REVIEW

**Comment by anmolbabu** [comment]
@r0h4n @shtripat @nthomas-redhat Please review

**Comment by anmolbabu** [comment]
Verified

**Comment by anmolbabu** [comment]
@r0h4n Done. Updated PR with pep8 fixes

File: tendrl/node_agent/monitoring/collectd/collectors/gluster/low_weight/tendrl_glusterfs_brick_disk_stats.py [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+                self.brick_details[
+                    'clusters.%s.volumes.%s.nodes.%s.bricks.%s.device.%s.'
+                    'disk_octets.read' % (
+                        self.CONFIG['integration_id'],
+                        vol_name,
+                        brick_host.replace('.', '_'),
+                        brick_path.replace('/', '|'),
+                        brick_device.replace('/dev/', '')
+                    )
+                ] = self.get_interval_disk_io_stat(
```
Comment (nthomas-redhat) [inline tendrl/node_agent/monitoring/collectd/collectors/gluster/low_weight/tendrl_glusterfs_brick_disk_stats.py]: @anmolbabu , why you are making these method calls again and again(same as in line 131). Instead you can make the call once use it at both places right? Is there specific reason why you are doing like this?

I think the same is applicable at multiple places in this method

File: tendrl/node_agent/monitoring/collectd/collectors/gluster/low_weight/tendrl_glusterfs_brick_disk_stats.py [hunk 2]
```diff
@@ -0,0 +1,421 @@
...
+
+    def get_brick_source_and_mount(self, brick_path):
+        # source and target correspond to fields "Filesystem" and
+        # "Mounted on" from df command output. The below command
+        # gives the filesystem and mount point for a given path,
+        # Eg: "/dev/mapper/tendrlMyBrick4_vg-tendrlMyBrick4_lv " \
+        #     "/tendrl_gluster_bricks/MyBrick4_mount"
+
+        command = "df --output=source,target " + brick_path
+        out, err = gluster_utils.exec_command(command)
```
Comment (shtripat) [inline tendrl/node_agent/monitoring/collectd/collectors/gluster/low_weight/tendrl_glusterfs_brick_disk_stats.py]: Cant we use `cmd_utils` from commons here?

File: tendrl/node_agent/monitoring/collectd/collectors/gluster/low_weight/tendrl_glusterfs_brick_disk_stats.py (hunk 1 above)
Comment (anmolbabu) [reply tendrl/node_agent/monitoring/collectd/collectors/gluster/low_weight/tendrl_glusterfs_brick_disk_stats.py]: There's not much gain right..
I mean the info even now is fetched only once but diff calculations are happening on demand

File: tendrl/node_agent/monitoring/collectd/collectors/gluster/low_weight/tendrl_glusterfs_brick_disk_stats.py (hunk 2 above)
Comment (anmolbabu) [reply tendrl/node_agent/monitoring/collectd/collectors/gluster/low_weight/tendrl_glusterfs_brick_disk_stats.py]: I thought about it but that would make the plugins tied to tendrl but now except for config the plugins are independent and don't have any dependency on any tendrl module

Comment (shtripat) [reply tendrl/node_agent/monitoring/collectd/collectors/gluster/low_weight/tendrl_glusterfs_brick_disk_stats.py]: Ack

**Pull request merged**